app/Model/Attribute.php

```diff
@@ -604,6 +616,22 @@ public function afterFind($results, $primary = false)
             if (isset($v['Attribute']['object_relation']) && $v['Attribute']['object_relation'] === null) {
                 $results[$k]['Attribute']['object_relation'] = '';
             }
+            if (!empty($v['Attribute']['first_seen'])) {
+                $fs = $results[$k]['Attribute']['first_seen'];
```

Comment:
It is possible to make function from this algo? Because the same implementation is copied four times.

Reply:
Good point.